tools/buildsteps/windows/BuildSetup.bat

```diff
@@ -41,10 +41,26 @@ FOR %%b in (%*) DO (
   IF %%b==sh SET useshell=sh
 )
 
-SET PreferredToolArchitecture=x64
-SET buildconfig=Release
-set WORKSPACE=%base_dir%\kodi-build.%TARGET_PLATFORM%
+IF DEFINED PreferredToolArchitecture (
+  echo Build Tool Arch: %PreferredToolArchitecture%
+) else (
+  echo Setting Default Tool Arch: %PROCESSOR_ARCHITECTURE%
+  SET PreferredToolArchitecture=%PROCESSOR_ARCHITECTURE%
+)
+
+IF DEFINED buildconfig (
+  echo Build Configuration: %buildconfig%
+) else (
+  echo Setting Default Build Configuration: Release
+  SET buildconfig=Release
+)
 
+IF DEFINED BUILDDIR (
+  echo BuildDir: %BUILDDIR%
+) else (
+  echo Setting Default Workspace: %base_dir%\kodi-build.%TARGET_PLATFORM%
+  SET WORKSPACE=%base_dir%\kodi-build.%TARGET_PLATFORM%
```

Comment:
```suggestion
  SET BUILDDIR=%base_dir%\kodi-build.%TARGET_PLATFORM%
```

In the last update this is wrong. Is creating the builds in `tools\buildsteps\windows` !!!!